README.md

```diff
@@ -62,214 +61,73 @@ Group Project Navigator is always displayed and provides quick access to most co
 * Private discussions - connect with teammates.
 * Ask Teaching Assistant - ask course staff for help in seconds.
 
-## Setup
+## High Level Overview
 
-In order to use Group Project XBlock v2 in a course, XBlock must be installed into LMS environment and enabled in 
-course's advanced settings. Also, Group Project XBlock uses a number of instance-specific configuration variables, so 
-setting them is considered a part of setup process.
+This is covered in more detail in [the authoring documentation](/docs/authoring.md). 
 
-### Installing XBlock into LMS
+### Project XBlock 
 
-If you're using edx-solutions/edx-platform fork, Group Project v2 XBlock is already installed. Otherwise, installing 
-Group Project XBlock v2 is done the very same way other XBlocks are installed:
+Top most component of Group Work v2 is Project XBlock. Project XBlock holds a number of activities. 
+Workgroups are defined on project level, i.e. student work with the same people thought all activities. Workgroups 
+are not defined inside GroupWorkProject XBlock, currently they must be defined in the proprietary Apros management 
+interface, XBlocks belonging to the project query `edx-solutions` api for the workgroups. 
 
-1. Open terminal (or shell) to the LMS instance.
-2. Switch to LMS virtualenv
-3. pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@ **version_hash** #egg=xblock-group-project-v2
+Apart from being a parent of some number of activities, group project should contain following children XBlocks: 
```

Comment:
Just had a look at the authoring docs. It says that the navigator xblock is required, but it doesn't say anything about the discussion xblock.